public/app/features/alerting/routes.tsx

```diff
@@ -168,10 +168,7 @@ export function getAlertingRoutes(cfg = config): RouteDescriptor[] {
     },
     {
       path: '/alerting/history/',
-      roles: evaluateAccess([
-        AccessControlAction.AlertingInstanceRead,
-        AccessControlAction.AlertingInstancesExternalRead,
-      ]),
+      roles: evaluateAccess([AccessControlAction.AlertingRuleRead]),
```

Comment:
we only need this permission as history is only available for Grafana managed rules